http4s/server/src/main/scala/endpoints4s/http4s/server/JsonEntities.scala

```diff
@@ -93,6 +93,6 @@ private object JsonEntities {
       endpoints: EndpointsWithCustomErrors
   )(encoder: Encoder[A, String]): endpoints.ResponseEntity[A] =
     EntityEncoder[endpoints.Effect, Chunk[Byte]]
-      .contramap[A](value => Chunk.array(encoder.encode(value).getBytes()))
+      .contramap[A](value => Chunk.array(encoder.encode(value).getBytes("UTF-8")))
```

Comment:
Would it be OK to use a predefined value instead of a string literal?

```suggestion
      .contramap[A](value => Chunk.array(encoder.encode(value).getBytes(http4s.Charset.`UTF-8`.nioCharset)))
```

Reply:
Yes, I agree that would be better.

Reply:
@hangilc, I think you didn’t push your changes

Reply:
Sorry. I pushed the changes.

Reply:
In the name of consistency with the rest of the project, shouldn't this be `StandardCharsets.UTF_8`? Functionally speaking, they're all equivalent.

Reply:
I’m happy with either :)